service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2082,9 +2075,7 @@ components:
         id:
           $ref: "#/components/schemas/CohortId"
         revisionId:
-          description: ID of the cohort revision, if frozen, null otherwise
```

Comment:
This seems useful to me, why remove?

Reply:
description field is allowed only for fields (properties) are are defined in place, not for references. 

in this case - revisionId: is reference to another custom schema type ($ref: "#/components/schemas/CohortRevisionId"). The definition of CohortRevisionId internally specifies the type of object and the description.

 in other words, removed it since it was a duplicate. The Swagger UI will still have the description from underlying referenced objects. 
 I can retain them if needed. let me know if so.

Reply:
I see, okay good to know. I think this comment applies to this property specifically, not necessarily the referenced object everywhere it's used. Perhaps we can refactor this as a comment inline here, rather than using the description field?

Reply:
Sure, I will mark them as comments in the same place

Reply:
Scratch the previous comment, saw you other comment on reviewing all properties. If there are no concerns we could review the properties and update the comments appropriately at the same time (constraints for example).

Reply:
Constraints are fine but I'd prefer to keep comments like this in that describe behavior. Otherwise we have to go re-discover the behavior later. Reformatting is fine, but removing information I'd like to avoid.

Reply:
Noted, thanks